src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -70,6 +71,11 @@ const ProgressiveProfiling = (props) => {
   }, [DASHBOARD_URL, registrationResponse]);
 
   useEffect(() => {
+    const queryParams = getAllPossibleQueryParams();
+    if (authenticatedUser?.userId && queryParams?.host) {
```

Comment:
do you think there should be a check for embedded too?

Reply:
We can add but there is no need to add this because `host` is also fullfling the same purpose.